ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_trigonometric_functions.py

```diff
@@ -73,6 +73,7 @@ def test_numpy_cos(
     ),
     native_array=helpers.array_bools(num_arrays=1),
 )
+
```

Comment:
There should not be a blankline here between the decorator and its corrisponding function.